src/content/ccip/tutorials/send-arbitrary-data-receipt-acknowledgment.mdx

```diff
@@ -0,0 +1,466 @@
+        style="max-width: 70%;"
+      />
+
+1. Open the [CCIP Explorer](https://ccip.chain.link/) and use the transaction hash you copied to search for your cross-chain transaction.
+
+   <ClickToZoom
+     src="/images/ccip/tutorials/ccip-explorer-sepolia-mumbai.webp"
+     alt="Chainlink CCIP Explorer - Sepolia to Mumbai Transaction Details"
+     style="max-width: 70%;"
+   />
+
+   After the transaction is finalized on the source chain, it will take a few minutes for CCIP to deliver the data to _Polygon Mumbai_ and call the `ccipReceive` function on your _acknowledger_ contract.
+
+1. Copy the message ID from the CCIP Explorer transaction details. You will use this message ID to track your message status on the _message tracker_ contract. In this example, it is `0x8773e27ea01eb0e7671da8d63e14a647713cb1274b7110141d249d3b8fce116b`.
+
+1. On the **Deploy & Run Transactions** tab in Remix, expand your _message tracker_ contract in the **Deployed Contracts** section.
+
+1. Paste the message ID you copied from the CCIP explorer as the argument in the **messagesInfo** getter function. Click **messagesInfo** to read the message status.
+
+   <ClickToZoom
+     src="/images/ccip/tutorials/message-tracker-message-status-1.webp"
+     alt="Chainlink CCIP - Message Tracker Get Message Status - 1"
+     style="max-width: 70%;"
+   />
+
+   Note the returned `status 1`. This value indicates that the _message tracker_ contract has updated your message status to the `Sent` status as defined by the `MessageStatus` `enum` in the _message tracker_ contract.
+
+   ```solidity
+   // Enum is used to track the status of messages sent via CCIP.
+   // `NotSent` indicates a message has not yet been sent.
+   // `Sent` indicates that a message has been sent to the Acknowledger contract but not yet acknowledged.
+   // `Confirmed` indicates that the Message Tracker contract has received the acknowledgment from the Acknowledger contract.
+   enum MessageStatus {
+     NotSent, // 0
+     Sent, // 1
+     Confirmed // 2
+   }
+   ```
+
+1. When the transaction is marked with a "Success" status on the [CCIP Explorer](https://ccip.chain.link/), the CCIP transaction and the destination transaction are complete. The _acknowledger_ contract has received the message from the _message tracker_ contract.
+
+   <ClickToZoom
+     src="/images/ccip/tutorials/ccip-explorer-sepolia-mumbai-success.webp"
+     alt="Chainlink CCIP Explorer - Sepolia to Mumbai Transaction Success"
+     style="max-width: 70%;"
+   />
+
+#### Acknowledgment message
+
+The `acknowledgePayLINK` function acts as a custom [`sendMessagePayLINK`](https://docs.chain.link/ccip/tutorials/send-arbitrary-data#sending-data-and-pay-in-link) function nested within the `ccipReceive` function of the _acknowledger_ contract and sends an acknowledgment (a CCIP message) to the _message tracker_ contract upon the initial message receipt. The data transferred in this acknowledgment message is the initial message ID.
```

Comment:
It's a bit confusing because you mix up code explanation with the tutorial. Users who haven't read the code explanation yet or the contract code will be confused. I suggest to rework this part:

- Just keep it general and inform that the ccipReceive function processes the message then sends a CCIP message (containing the acknowledgment) to the sender
- When the message is sent, an event is sent (indeed, you can keep the structure of the event because it is necessary)
- Link to the code explanation for more details